pkg/obfuscator/domain.go

```diff
@@ -4,18 +4,22 @@ import (
 	"fmt"
 	"regexp"
 	"strings"
+
+	"github.com/openshift/must-gather-clean/pkg/schema"
 )
 
 const (
-	domainPattern      = `([a-zA-Z0-9\.]*\.)?(%s)`
-	obfuscatedTemplate = "domain%07d"
+	domainPattern           = `([a-zA-Z0-9\.]*\.)?(%s)`
+	obfuscatedTemplate      = "domain%07d"
+	staticDomainReplacement = "example-domain.com"
```

Comment:
shall we just generate `x`.repeat(n)+ ".com"? I think example-domain might be a bit confusing

Reply:
As we are referring to the static replacement, what is the significance of "n" here?